src/it/scala/org/zalando/spearheads/innkeeper/routes/GetDeletedRoutesSpec.scala

```diff
@@ -0,0 +1,84 @@
+package org.zalando.spearheads.innkeeper.routes
+
+import java.time.LocalDateTime
+
+import akka.http.scaladsl.model.StatusCodes
+import org.scalatest.{Matchers, BeforeAndAfter, FunSpec}
+import org.zalando.spearheads.innkeeper.api.{RouteName, RouteOut}
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecsHelper._
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecTokens._
+import org.zalando.spearheads.innkeeper.routes.RoutesRepoHelper._
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+import org.zalando.spearheads.innkeeper.api.JsonProtocols._
+
+/**
+ * @author Alexey Venderov
+ */
+class GetDeletedRoutesSpec extends FunSpec with BeforeAndAfter with Matchers {
+
+  describe("get /deleted-routes") {
+    describe("success") {
+
+      before {
+        recreateSchema
+      }
+
+      it("should return an empty collection if there are no deleted routes") {
+        insertRoute("R1")
+        insertRoute("R2")
+
+        val response = getDeletedRoutes(LocalDateTime.now(), READ_TOKEN)
+        response.status shouldBe StatusCodes.OK
+
+        val entity = entityString(response)
+
+        val deletedRoutes = entity.parseJson.convertTo[Seq[RouteOut]]
+        deletedRoutes.size shouldBe 0
```

Comment:
again, in this file please replace all `shouldBe` with `should be`, for consistency with other tests.